test/language/rest-parameters/arrow-function.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 14.1
+description: >arrow functions
+includes: [compareArray.js]
+---*/
+var fn = (a, b, ...c) => c;
+
+assert(compareArray(fn(), []), "`compareArray(fn(), [])` returns `true`");
```

Comment:
Ugh, we probably need an assert.arrayElementsSameValue or somesuch. This assert(compareArray( boilerplate really grates on me. (Ok for now though)